modules/aws/vpc_test.go

```diff
@@ -17,6 +21,37 @@ func TestGetDefaultVpc(t *testing.T) {
 	assert.Regexp(t, "^vpc-[[:alnum:]]+$", vpc.Id)
 }
 
+func TestGetVpcById(t *testing.T) {
+	t.Parallel()
+
+	region := GetRandomStableRegion(t, nil, nil)
+	vpc := createVpc(t, region)
+	defer deleteVpc(t, *vpc.VpcId, region)
+
+	vpcTest := GetVpcById(t, *vpc.VpcId, region)
+	assert.Equal(t, *vpc.VpcId, vpcTest.Id)
+}
+
+func TestGetVpcsE(t *testing.T) {
+	t.Parallel()
+
+	region := GetRandomStableRegion(t, nil, nil)
+	azs := GetAvailabilityZones(t, region)
+
+	var isDefaultFilterName = "isDefault"
+	var isDefaultFilterValue = "true"
+
+	defaultVpcFilter := ec2.Filter{Name: &isDefaultFilterName, Values: []*string{&isDefaultFilterValue}}
+	vpcs, _ := GetVpcsE(t, []*ec2.Filter{&defaultVpcFilter}, region)
+
+	assert.Equal(t, len(vpcs), 1)
```

Comment:
This should be `require.Equal` so that the line below doesn't `panic` when there are no VPCs.

Reply:
To clarify, `assert` marks the test as `failed` when the assertion fails, but the test continues. `require` does the same thing and also halts the execution of the function.